TSC-Charter.md

```diff
@@ -74,9 +74,50 @@ including:
 * Development process and any coding standards.
 * Mediating technical conflicts between Collaborators or Foundation
   projects.
+* Establishing guidelines for communication by project contributors about the
+  technical development processes of the project, releases and release announcements,
+  technical documentation, contributor onboarding and conduct, project governance,
+  and technical direction.
 
 The TSC will define Node.js project’s release vehicles.
 
+### Exclusions and limitations of TSC responsibilities
+
+The TSC and the project is not permitted to directly, or indirectly, solicit
+funds or financial sponsorship for the Node.js project itself; or to enter into
+any financial agreements on behalf of the project. Such activities remain the
+sole responsibility of the OpenJS Foundation. The TSC may, however, actively
+seek technical or other non-financial sponsorship for the project, such as
+donation of needed services, hardware, software, or other resources, provided
+that such sponsorship does not create a conflict of interest with the OpenJS
+Foundation or its other projects, or otherwise violate Foundation policies.
+
+Specific exclusions from the communication oversight responsibilities include:
+
+ * Brand marketing
+ * Fundraising and financial sponsorship of the project (including, but not
+   limited to, the OpenJS Foundation's own financial sponsorship of the project)
+ * Legal matters
+
+All communication from the project touching on these topics must be reviewed
+and approved by the Foundation.
+
+All uses of OpenJS Foundation-owned trademarks and intellectual property are,
+subject to the [OpenJS Foundation Trademark Policy](https://trademark-policy.openjsf.org/).
+This includes use of the marks -- in full or in part -- in domain names, social
+media account handles, event names and other similar identifiers.
+
+The TSC may establish and administer project-specific communication channels,
+such as social media accounts and discussion forums, administered under the
+oversight of the TSC, subject to the above requirements. Should such channels
+make any use of OpenJS Foundation-owned trademarks or intellectual property, the
+channels must be capable of being administered jointly by both the TSC and the
+Foundation.
+
+Nothing in this charter should be interpreted as preventing the Foundation
+from also publishing content on any channel under its control about any aspect
+of the project.
```

Comment:
I personally don't see anything Node. js-specific here, and I think this should be included in the CPC charter.

cc @tobie

Reply:
Yeah, charter and/or just policies.

Reply:
I'm not sure charter amendments are the best vehicle for internal communication issues, tbh. (I agree that there are necessary changes to the comms/marketing guidelines, though, but I'm not even sure those will require charter changes.) We need to get people on the same page and understand it's in everyone's interested to find common ground.